.github/workflows/publish-charm.yaml

```diff
@@ -0,0 +1,29 @@
+name: publish-charm
+
+on:
+  workflow_call:
+    secrets:
+      CHARMCRAFT_AUTH:
+        required: true
+
+jobs:
+  publish-charm:
+    runs-on: ubuntu-22.04
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Install charmcraft
+        run: sudo snap install charmcraft --classic
+      - name: Fetch Tested Charm
+        uses: actions/download-artifact@v3
+        with:
+          name: tested-charm
+      - name: Move charm in current directory
+        run: find ./ -name sdcore-smf_ubuntu-22.04-amd64.charm -exec mv -t ./ {} \;
+      - name: Select Charmhub channel
+        uses: canonical/charming-actions/channel@2.2.5
+        id: channel
+      - name: Upload charm to Charmhub
```

Comment:
Please use the new workflow. Ref [here](https://github.com/canonical/sdcore-upf-operator/blob/main/.github/workflows/publish-charm.yaml#L27)

Reply:
Done.